content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -40,13 +51,14 @@ An example of the request format:
 ```sh
 #Syntax: server_info
 rippled server_info
+rippled server_info counters # for optional performance statistics
```

Comment:
Instead of adding a second line, should update the syntax line above, maybe with a note like the ones on [`ledger`'s Commandline syntax](http://xrpl.local/ledger.html)

Reply:
fixed in the latest tip